content/en/docs/concepts/signals/traces.md

```diff
@@ -298,7 +298,7 @@ Each span has a status. The three possible values are:
 The default value is `Unset`. A span status that is `Unset` means that the
 operation it tracked successfully completed without an error.
 
-When a span status is `Error`, then that means some error ocurred in the
```

Comment:
@svrnm - we should find out why cSpell doesn't detect this. I made some quick tests but wasn't able to find out why. Maybe a bug with the its dictionary?

Reply:
that's odd indeed, they even have it here:

https://github.com/streetsidesoftware/cspell/blob/3914546fdd862809df2ebf466e502adfbfd93384/rfc/rfc-0001%20suggestions/fixtures/EnglishTypos_common.typo.txt#L2584

Reply:
https://github.com/streetsidesoftware/cspell-dicts/blob/dc56f6f63e68134c012456f796ef5f67d745ff06/dictionaries/en_US/src/en_US.txt#L13385 ... ❓

Reply:
raised an issue: https://github.com/streetsidesoftware/cspell-dicts/issues/2940